 2017-01-18 16:43:59,872 [P6256/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.5.7 application starting on DESKTOP-MRCBO5I
 2017-01-18 16:43:59,921 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2017-01-18 16:43:59,946 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 24ms)
 2017-01-18 16:44:00,063 [P6256/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2017-01-18 16:44:00,071 [P6256/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2017-01-18 16:44:00,081 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of umbraco.interfaces.IApplicationStartupHandler
 2017-01-18 16:44:00,688 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.IApplicationStartupHandler, found 0 (took 607ms)
 2017-01-18 16:44:00,708 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2017-01-18 16:44:00,720 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter, found 0 (took 11ms)
 2017-01-18 16:44:00,721 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2017-01-18 16:44:00,733 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyValueConverter, found 0 (took 12ms)
 2017-01-18 16:44:00,759 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.Mvc.SurfaceController
 2017-01-18 16:44:00,772 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.Mvc.SurfaceController, found 0 (took 12ms)
 2017-01-18 16:44:00,773 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.WebApi.UmbracoApiController
 2017-01-18 16:44:00,789 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.WebApi.UmbracoApiController, found 0 (took 16ms)
 2017-01-18 16:44:00,968 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IThumbnailProvider
 2017-01-18 16:44:00,981 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IThumbnailProvider, found 0 (took 13ms)
 2017-01-18 16:44:00,983 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IImageUrlProvider
 2017-01-18 16:44:01,000 [P6256/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IImageUrlProvider, found 0 (took 17ms)
 2017-01-18 16:44:03,777 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:03,784 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:04,074 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:04,075 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:05,117 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:05,118 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:06,160 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:06,161 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:07,203 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:07,204 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:08,233 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:08,234 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:09,269 [P6256/D2/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:09,270 [P6256/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:09,276 [P6256/D2/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but the Umbraco cannot connect to the database.
   at Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   at Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   at Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2017-01-18 16:44:17,712 [P6256/D3/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.5.7 application starting on DESKTOP-MRCBO5I
 2017-01-18 16:44:17,743 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2017-01-18 16:44:17,759 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 14ms)
 2017-01-18 16:44:17,821 [P6256/D3/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2017-01-18 16:44:17,837 [P6256/D2/T11] INFO  Umbraco.Core.MainDom - Stopping...
 2017-01-18 16:44:17,837 [P6256/D2/T11] INFO  Umbraco.Core.MainDom - Released MainDom.
 2017-01-18 16:44:17,884 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.IApplicationStartupHandler, found 0 (took 24ms)
 2017-01-18 16:44:17,921 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2017-01-18 16:44:17,922 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter, found 0 (took 1ms)
 2017-01-18 16:44:17,923 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2017-01-18 16:44:17,924 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyValueConverter, found 0 (took 1ms)
 2017-01-18 16:44:17,940 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.Mvc.SurfaceController
 2017-01-18 16:44:17,940 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.Mvc.SurfaceController, found 0 (took 2ms)
 2017-01-18 16:44:17,940 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.WebApi.UmbracoApiController
 2017-01-18 16:44:17,955 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.WebApi.UmbracoApiController, found 0 (took 12ms)
 2017-01-18 16:44:18,033 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IThumbnailProvider
 2017-01-18 16:44:18,033 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IThumbnailProvider, found 0 (took 2ms)
 2017-01-18 16:44:18,033 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IImageUrlProvider
 2017-01-18 16:44:18,033 [P6256/D3/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IImageUrlProvider, found 0 (took 1ms)
 2017-01-18 16:44:19,551 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:19,560 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:19,766 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:19,767 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:20,798 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:20,814 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:21,845 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:21,861 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:22,908 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:22,908 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:23,970 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:23,986 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:25,048 [P6256/D3/T1] WARN  Umbraco.Core.Persistence.DbConnectionExtensions - Configured database is reporting as not being available! DataSource: |DataDirectory|\Umbraco.sdf. Exception: System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.DbConnectionExtensions.IsAvailable(IDbConnection connection)
 2017-01-18 16:44:25,048 [P6256/D3/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = False
 2017-01-18 16:44:25,048 [P6256/D3/T1] ERROR Umbraco.Core.UmbracoApplicationBase - An unhandled exception occurred
Umbraco.Core.Exceptions.UmbracoStartupFailedException: Umbraco cannot start. A connection string is configured but the Umbraco cannot connect to the database.
   at Umbraco.Core.CoreBootManager.EnsureDatabaseConnection()
   at Umbraco.Core.CoreBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Web.WebBootManager.Complete(Action`1 afterComplete)
   at Umbraco.Core.UmbracoApplicationBase.StartApplication(Object sender, EventArgs e)
   at Umbraco.Core.UmbracoApplicationBase.Application_Start(Object sender, EventArgs e)
 2017-01-18 16:44:35,057 [P6256/D3/T11] INFO  Umbraco.Core.MainDom - Stopping...
 2017-01-18 16:44:35,057 [P6256/D3/T11] INFO  Umbraco.Core.MainDom - Released MainDom.
 2017-01-18 16:44:35,276 [P6256/D3/T11] ERROR UmbracoExamine.DataServices.UmbracoContentService - EXCEPTION OCCURRED reading GetAllUserPropertyNames
System.Data.SqlServerCe.SqlCeException (0x80004005): The database file cannot be found. Check the path to the database. [ Data Source = E:\Zmartforce\Umbraco Test\testApplicationUmbraco\testApplicationUmbraco\App_Data\Umbraco.sdf ]
   at System.Data.SqlServerCe.SqlCeConnection.Open(Boolean silent)
   at System.Data.SqlServerCe.SqlCeConnection.Open()
   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.<>c__DisplayClass1.<ExecuteAction>b__0()
   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   at Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction(Action action)
   at Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection, RetryPolicy retryPolicy)
   at Umbraco.Core.Persistence.PetaPocoConnectionExtensions.OpenWithRetry(IDbConnection connection)
   at Umbraco.Core.Persistence.Database.OpenSharedConnection()
   at Umbraco.Core.Persistence.Database.<Query>d__7`1.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at UmbracoExamine.DataServices.UmbracoContentService.GetAllUserPropertyNames()
 2017-01-18 16:49:49,213 [P7464/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.5.7 application starting on DESKTOP-MRCBO5I
 2017-01-18 16:49:49,255 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2017-01-18 16:49:49,279 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 24ms)
 2017-01-18 16:49:49,402 [P7464/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2017-01-18 16:49:49,415 [P7464/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2017-01-18 16:49:49,428 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of umbraco.interfaces.IApplicationStartupHandler
 2017-01-18 16:49:49,931 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.IApplicationStartupHandler, found 0 (took 503ms)
 2017-01-18 16:49:49,950 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2017-01-18 16:49:49,956 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter, found 0 (took 5ms)
 2017-01-18 16:49:49,957 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2017-01-18 16:49:49,972 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IPropertyValueConverter, found 0 (took 15ms)
 2017-01-18 16:49:49,999 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.Mvc.SurfaceController
 2017-01-18 16:49:50,009 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.Mvc.SurfaceController, found 0 (took 9ms)
 2017-01-18 16:49:50,009 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.WebApi.UmbracoApiController
 2017-01-18 16:49:50,021 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.WebApi.UmbracoApiController, found 0 (took 11ms)
 2017-01-18 16:49:50,148 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IThumbnailProvider
 2017-01-18 16:49:50,159 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IThumbnailProvider, found 0 (took 11ms)
 2017-01-18 16:49:50,160 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Media.IImageUrlProvider
 2017-01-18 16:49:50,170 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Media.IImageUrlProvider, found 0 (took 10ms)
 2017-01-18 16:49:52,324 [P7464/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2017-01-18 16:49:52,936 [P7464/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2017-01-18 16:49:52,959 [P7464/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2017-01-18 16:49:53,118 [P7464/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2017-01-18 16:49:53,243 [P7464/D2/T1] WARN  Umbraco.Core.Sync.DatabaseServerMessenger - No last synced Id found, this generally means this is a new server/install. The server will build its caches and indexes, and then adjust its last synced Id to the latest found in the database and maintain cache updates based on that Id.
 2017-01-18 16:49:53,308 [P7464/D2/T1] INFO  umbraco.content - Load Xml from file...
 2017-01-18 16:49:53,336 [P7464/D2/T1] INFO  umbraco.content - Loaded Xml from file.
 2017-01-18 16:49:53,347 [P7464/D2/T1] INFO  umbraco.content - Loading content from database...
 2017-01-18 16:49:55,480 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.Persistence.Mappers.BaseMapper
 2017-01-18 16:49:55,492 [P7464/D2/T1] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.Persistence.Mappers.BaseMapper, found 0 (took 12ms)
 2017-01-18 16:49:55,869 [P7464/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 6952ms)
 2017-01-18 16:49:56,301 [P7464/D2/T6] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://localhost:50285/umbraco (UmbracoModule request)
 2017-01-18 16:49:56,853 [P7464/D2/T6] WARN  Umbraco.Web.UmbracoModule - Umbraco has no content
 2017-01-18 16:49:57,586 [P7464/D2/T12] INFO  umbraco.content - Save Xml to file...
 2017-01-18 16:49:57,606 [P7464/D2/T12] INFO  umbraco.content - Saved Xml to file.
 2017-01-18 16:50:10,255 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Core.PropertyEditors.IParameterEditor
 2017-01-18 16:50:10,325 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Core.PropertyEditors.IParameterEditor, found 0 (took 69ms)
 2017-01-18 16:50:18,508 [P7464/D2/T7] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: Login attempt succeeded for username stefan from IP address ::1
 2017-01-18 16:50:18,508 [P7464/D2/T7] INFO  Umbraco.Core.Security.BackOfficeSignInManager - Event Id: 0, state: User: stefan logged in from IP address ::1
 2017-01-18 16:50:19,180 [P7464/D2/T7] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.Trees.TreeController
 2017-01-18 16:50:19,198 [P7464/D2/T7] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.Trees.TreeController, found 0 (took 17ms)
 2017-01-18 16:50:19,985 [P7464/D2/T13] INFO  Umbraco.Core.PluginManager - Starting resolution types of umbraco.interfaces.IApplication
 2017-01-18 16:50:20,011 [P7464/D2/T13] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.IApplication, found 0 (took 25ms)
 2017-01-18 16:50:20,576 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Starting resolution types of Umbraco.Web.Trees.TreeController
 2017-01-18 16:50:20,576 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Completed resolution of types of Umbraco.Web.Trees.TreeController, found 0 (took 0ms)
 2017-01-18 16:50:20,582 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Starting resolution types of umbraco.interfaces.ITree
 2017-01-18 16:50:20,617 [P7464/D2/T20] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.ITree, found 0 (took 35ms)
 2017-01-18 16:50:21,041 [P7464/D2/T7] INFO  Umbraco.Core.PluginManager - Starting resolution types of umbraco.interfaces.ITree
 2017-01-18 16:50:21,080 [P7464/D2/T7] INFO  Umbraco.Core.PluginManager - Completed resolution of types of umbraco.interfaces.ITree, found 0 (took 38ms)
